### PR TITLE
sprint challenge1

### PR DESCRIPTION
**Pull request by aleksandr-ardanov**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,14 +34,27 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
+every semantic html tag has it's meaning. it makes it easier for us to markup a page and for other to read our code
+
```
Comment (Elisa-Alvarez) [inline README.md:38]: Very true, great answer. Another reason is accessibility, in some tech fields lack of semantic code can lead to accessibility legal action here is a great resource for more info ->https://developer.mozilla.org/en-US/docs/Learn/Accessibility/HTML

File: about.html [hunk 2]
```diff
@@ -7,59 +7,118 @@
...
+
+        <a href="#">Learn More</a>
+    </div>
+
+  </div>
+
+</section>
+
+<div class="sometextbottom">
+
```
Comment (Elisa-Alvarez) [inline about.html:111]: For this div could it have been changed to a more semantic portion of code like a section or an article?

**Review by Elisa-Alvarez — COMMENTED** [review]
Your work is very impressive, nice job using :nth-child selector.

**Comment by baraboo312** [comment]
yes, it can be replaced by section element

File: about.html (hunk 2 above)
Comment (baraboo312) [reply about.html:111]: yes, it can be replaced by  a section element

**Pull request closed without merging**